load.go

```diff
@@ -447,3 +453,57 @@ func (b *ArtifactBuild) processBuildArgs(lex *shell.Lex, args map[string]string,
 
 	return goerrors.Join(errs...)
 }
+
+func (img *ImageConfig) processBuildArgs(lex *shell.Lex, args map[string]string, allowArg func(string) bool) error {
+	var errs error
+	for _, p := range []*string{&img.Base, &img.Cmd, &img.User, &img.Entrypoint, &img.StopSignal, &img.WorkingDir} {
+		updated, err := expandArgs(lex, *p, args, allowArg)
+		if err != nil {
+			errs = goerrors.Join(errs, errors.Wrap(err, "imgconfig"))
+		}
+		*p = updated
+	}
+
+	for i, s := range img.Env {
```

Comment:
I'm a little hesitant support this here but not against it.

Reply:
:+1: